dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -244,7 +244,7 @@ const shouldShowContributor = (format: ArticleFormat) => {
 	}
 };
 
-const AvatarContainer = ({ children }: { children: React.ReactNode }) => (
+const MetaAvatarContainer = ({ children }: { children: React.ReactNode }) => (
```

Comment:
I renamed this to avoid confusion with the other component also called `AvatarContainer`